src/main/java/org/skriptlang/skript/elements/displays/expressions/ExprDisplayTransformationRotation.java

```diff
@@ -0,0 +1,103 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.elements.displays.expressions;
+
+import org.bukkit.entity.Display;
+import org.bukkit.event.Event;
+import org.bukkit.util.Transformation;
+import org.jetbrains.annotations.Nullable;
+import org.joml.Quaternionf;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Display Transformation Rotation")
+@Description({
+        "Returns or changes the transformation rotation of <a href='classes.html#display'>displays</a>.",
+        "The left rotation is applied first, with the right rotation then being applied based on the rotated axis."
+})
+@Examples("set left transformation rotation of last spawned block display to quaternionf(1, 0, 0, 0) # reset block display")
+@Since("INSERT VERSION")
+public class ExprDisplayTransformationRotation extends SimplePropertyExpression<Display, Quaternionf> {
```

Comment:
Is it possible to make `transformation` optional in this expression? It seems unnecessarily wordy right now.

Reply:
I don't like that since rotation could mean multiple things when referring to a Display.

Reply:
What other things? I can't think of any off the top of my head, but i haven't really touched item display so maybe something in there?